code/mob/living/carbon/human.dm

```diff
@@ -1834,6 +1834,11 @@
 		hud.set_visible(hud.twohandr, 1)
 		hud.remove_item(I)
 		hud.add_object(I, HUD_LAYER+2, hud.layouts[hud.layout_style]["twohand"])
+		var/regex/offset = new(@"^icons/obj/(\d+)x\d+.dmi$")
+		if(offset.Find("[I.icon]"))
+			var/regex/off2 = new(@"^(CENTER[+-]\d:)(\d+)(.*)$")
```

Comment:
could you document this cursed behemoth so people know what it does if they can't read through it?